html/semantics/forms/the-label-element/clicking-interactive-content.html

```diff
@@ -16,6 +16,7 @@
   <input>
   <label>label</label>
   <object usemap=""></object>
+  <object></object>
```

Comment:
I think we could investigate making object interactive, if there's implementer interest, in a separate change. In the meantime, I think you probably want to remove both `<object usemap=""> and `<object>` from here.

Reply:
I agree with this comment, and I have a [chromium CL](https://chromium-review.googlesource.com/c/chromium/src/+/2725443) in progress with just this change, plus one more suggested by @domenic, to clicking-noninteractive-unlabelable-content.html.

Reply:
That would reduce coverage as this is the only test that found out this was a thing to begin with. Not entirely sure how to restructure it though. Or are you suggesting that because Chrome will add back some kind of coverage?

Reply:
I see, https://github.com/web-platform-tests/wpt/commit/1844f17ecf2dd024c14cff75d21c9771084a6297 by @mfreed7 does indeed seem to do that. Right?

Reply:
@mfreed7 how about you import all this and land it as part of the Chromium change?

Reply:
@annevk, sure, I'm happy to. CL in progress [here](https://chromium-review.googlesource.com/c/chromium/src/+/2729397).

Reply:
Ok, should land soon:

https://github.com/web-platform-tests/wpt/pull/27864